cypress/integration/cookieBannerWS.js

```diff
@@ -13,8 +15,12 @@ describe('World Service Cookie Banner Test', () => {
         `${config[serviceName].assets.cookieAgreement}`,
       );
     });
+  });
+});
 
-    it(`should load the relevant translations for ${serviceName} (AMP)`, () => {
+describeForEuOnly('World Service Cookie Banner Transations - AMP', () => {
```

Comment:
The issue was that AMP cookie banner will only show within the EU, so have added this describe block.  These tests will run on our Jenkins CI (within EU) and be skipped on Travis CI (outside of EU).